components/app/footer/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020-2022 SubQuery Pte Ltd authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, Typography } from '../../common';
+import * as React from 'react';
+import styles from './Footer.module.css';
+interface LinksProps {
+  link: string;
+  icon?: string;
+  alt?: string;
+}
+
+const links: LinksProps[] = [
+  { link: 'mailto:hello@subquery.network', icon: '/icons/icon_1.svg' },
+  { link: 'https://twitter.com/subquerynetwork', icon: '/icons/icon_3.svg' },
+  { link: 'https://medium.com/@subquery', icon: '/icons/icon_4.svg' },
+  { link: 'https://github.com/OnFinality-io/subql', icon: '/icons/icon_5.svg' },
+  { link: 'https://youtube.com/c/SubQueryNetwork', icon: '/icons/icon_6.svg' },
+  { link: 'https://t.me/subquerynetwork', icon: '/icons/icon_7.svg' },
+  { link: 'https://www.linkedin.com/company/subquery', icon: '/icons/icon_8.svg' },
```

Comment:
[Suggestion] Please rename the svg assets align with its meaning, for example `email.svg` `twitter.svg`